src/lib.rs

```diff
@@ -37,6 +37,7 @@ use language::{
     SupportedLanguageName,
 };
 use plugin::get_loaded_filter;
+pub use plugin::PluginInitializeReturn;
```

Comment:
So reading this this is now "surface area for `tree_sitter_grep`-the-crate"?

I'm remembering that in some of those FFI-plugin-examples they were extracting this type of thing out into a shared "plugin" crate, maybe that is a good idea (I'll add an issue)?